src/mainboard/clevo/tgl-u/ramstage.c

```diff
@@ -30,6 +30,10 @@ struct smfi_cmd_set_fan_curve {
 	struct fan_curve curve;
 } __packed;
 
+struct smfi_cmd_camera_enablement_set {
+    bool enable;
+} __packed;
```

Comment:
Does this need to be `__packed`?

Reply:
Single element does not even need a structure...